src/rcl_bindings.cpp

```diff
@@ -1519,6 +2162,33 @@ BindingMethod binding_methods[] = {
     {"countPublishers", CountPublishers},
     {"countSubscribers", CountSubscribers},
     {"serviceServerIsAvailable", ServiceServerIsAvailable},
+    {"actionCreateClient", ActionCreateClient},
```

Comment:
I was considering moving all action node bindings to separate file (e.g. rcl_action_bindings.cpp) as this file is massive already.

Any preference @minggangw?